features/measurements/impl/src/test/java/org/opennms/netmgt/measurements/filters/impl/LoadDataHWForecastTest.java

```diff
@@ -0,0 +1,165 @@
+/*******************************************************************************
```

Comment:
need to update copyright headers here as well

Reply:
I see, I've been quite busy, I'll work it!